tests/bootstrap.php

```diff
@@ -1,3 +1,5 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
The ```declare(strict_types=1);``` is added in the every related ```Tests``` methods currently.
This bootstrap file is seems to be useless and it should change this approach to adding the ```vendor/autload.php``` in ```bootstrap``` attribute for the phpunit tag in ```phpunit.xml.dist``` file.

Here is the example:

```
<phpunit bootstrap="./vendor/autoload.php" colors="true">
```

Reply:
Agree. I will make that change.